packages/grafana-prometheus/src/querybuilder/components/LabelFilters.tsx

```diff
@@ -5,19 +5,21 @@ import { useEffect, useState } from 'react';
 
 import { SelectableValue } from '@grafana/data';
 import { EditorField, EditorFieldGroup, EditorList } from '@grafana/experimental';
-import { InlineFieldRow, InlineLabel } from '@grafana/ui';
+import { config } from '@grafana/runtime';
+import { ComboboxOption, InlineFieldRow, InlineLabel } from '@grafana/ui';
 
 import { QueryBuilderLabelFilter } from '../shared/types';
 
 import { LabelFilterItem } from './LabelFilterItem';
+import { LabelFilterItemCombobox } from './LabelFilterItemCombobox';
 
 export const MISSING_LABEL_FILTER_ERROR_MESSAGE = 'Select at least 1 label filter (label and value)';
 
 export interface LabelFiltersProps {
   labelsFilters: QueryBuilderLabelFilter[];
   onChange: (labelFilters: Array<Partial<QueryBuilderLabelFilter>>) => void;
-  onGetLabelNames: (forLabel: Partial<QueryBuilderLabelFilter>) => Promise<SelectableValue[]>;
-  onGetLabelValues: (forLabel: Partial<QueryBuilderLabelFilter>) => Promise<SelectableValue[]>;
```

Comment:
`ComboboxOption` is a stricter version of `SelectableValue`, so `ComboboxOption` is compatible with `SelectableValue`, but not the other way around.